lotus-soup/init.go

```diff
@@ -15,8 +15,8 @@ import (
 )
 
 func init() {
-	build.BlockDelaySecs = 2
-	build.PropagationDelaySecs = 1
+	// build.BlockDelaySecs = 2
+	// build.PropagationDelaySecs = 1
```

Comment:
Need to sort this out. Synchronised mining doesn't need to reduce the block delay and propagation delay.